spk/mediainfo/Makefile

```diff
@@ -0,0 +1,35 @@
+SPK_NAME = mediainfo
+SPK_VERS = 0.7.72
+SPK_REV = 1
+SPK_ICON = src/mediainfo.png
+
+DEPENDS  = cross/$(SPK_NAME)
+
+RELOAD_UI = yes
+STARTABLE = no
+
+MAINTAINER = cytec
```

Comment:
@Diaoul Do we have a general approach on the maintainer content? `SynoCommunity/maintainer`, `SynoCommunity`, or is `maintainer` accepted too?
Keep in mind that Package Center probably adds a linebreak without hyphenation in the middle of `maintainer` with the first option.